run.sh

```diff
@@ -9,13 +9,13 @@ run__prod () {
 run__dev () {
 	# dev mode
 	docker build --file Dockerfile.dev -t root-pythia-dev:latest .
-	docker run --rm --interactive --tty --env-file .env.dev --volume ./src:/opt/root-pythia/src root-pythia-dev:latest
+	docker run --rm --interactive --tty --env-file .env.dev --volume `realpath ./src`:/opt/root-pythia/src root-pythia-dev:latest
```

Comment:
What are these `realpath` here?

Reply:
Just because on my distro, for some reason, docker has to be given absolute paths, so doing it like this shouldn't disturb anyone's docker but makes mine happy 😄

Reply:
Ok!

Reply:
> I just [discovered](https://unix.stackexchange.com/questions/27428/what-does-backquote-backtick-mean-in-commands) the use of backquotes/backticks in bash actually ahah so thanks for that. But we maybe should use `$()` subshells instead of backticks because [backticks or not POSIX](https://stackoverflow.com/questions/9449778/what-is-the-benefit-of-using-instead-of-backticks-in-shell-scripts) (even if it is not a big thing here it's good coding habits in my opinion!)
> 
> But I read a bit about `realpath` and it seems to be shipped with a package, shouldn't we use `pwd -P` that seems equivalent instead? https://unix.stackexchange.com/questions/101080/realpath-command-not-found https://unix.stackexchange.com/questions/285916/real-current-directory

Fixed it thanks for the details. I discovered the backticks myself in a ctf :wink: